src/Randomizer.SMZ3.Tracking/Configuration/IMergeableConfig.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Randomizer.SMZ3.Tracking.Configuration.ConfigFiles;
+using Randomizer.SMZ3.Tracking.Configuration.ConfigTypes;
+
+namespace Randomizer.SMZ3.Tracking.Configuration
+{
+    public interface IMergeableConfig
```

Comment:
Like my previous comment, I'm not sure "Config" is actually all that relevant to what you're describing in the interface.